src/workspaces/workspaces.ts

```diff
@@ -0,0 +1,26 @@
+import { getWorkspaces } from "./getWorkspaces";
+
+const cache = new Map<string, string[]>();
+
+/**
+ * Get paths to every package.json in the workspace, given a cwd
+ * @param cwd
+ */
+export function getAllPackageJsonFiles(cwd: string) {
+  if (cache.has(cwd)) {
+    return cache.get(cwd);
+  }
+
+  const workspaces = getWorkspaces(cwd);
+  const packageJsonFiles = workspaces.map(
+    (workspace) => workspace.packageJson.packageJsonPath
+  );
```

Comment:
This file originally contained logic to identify workspaces and a multimatch to query `package.json` paths (?). Since this data is now part of workspaces, I'm now just doing this simple map operation.

Reply:
I believe this works in my use case of this